### ISSUE TITLE
Switch in dropdown.blade.php not updated in 5.1.3

### ISSUE THREAD

**Issue by RomainGoncalves** [laravel/jetstream#1519]
### Jetstream Version

5.1.3

### Jetstream Stack

Livewire

### Laravel Version

11

### PHP Version

8.2

### Database Driver & Version

MySQL

### Description

Today, I've tried to install a new Laravel using `laravel new app`
When registering a user an being sent to the dashboard, I encountered the error: Unhandled match case '60'

It says in release 5.1.3 that there is a fix for it, however when looking here, it seems it isn't part of the 5.1.3 tag:
https://github.com/laravel/jetstream/blob/v5.1.3/stubs/livewire/resources/views/components/dropdown.blade.php
I also downloaded the .zip source file for 5.1.3 to check, and the width 60 and default cases aren't in it

Am I doing something wrong?

### Steps To Reproduce

laravel new app
With Jetstream, Teams, Livewire, Pest
Register new user with the form
Error on dashboard page

**Comment by RomainGoncalves** [comment]
I looked more carefully since I posted, and actually, where I found the fix on the repo was in the NEW commits SINCE the release. 
Sorry for the misunderstanding. On the release page, there's a link that says "[3 commits ](https://github.com/laravel/jetstream/compare/v5.1.3...5.x)to 5.x since this release", keyword: since.... that's where I got the info.

I guess the issue still remains for now. For a new install with Teams, the switch doesn't have the value 60 or a default case in 5.1.3
